src/code/code_800EC960.c

```diff
@@ -155,6 +155,9 @@ u8 sSeqModeInput = 0;
 #define SEQ_FLAG_SKIP_HARP_INTRO (1 << 6)
 #define SEQ_FLAG_NO_AMBIENCE (1 << 7)
 
+#define SEQ_ID(packedSeqId) ((packedSeqId)&0xFF)
```

Comment:
```suggestion

#define SEQ_ID(packedSeqId) ((packedSeqId) & 0xFF)
```

Reply:
For macros like this that are in the .c file, the formatter will just squeeze this back together again and remove the whitespace

Reply:
I didnt realize it wasnt in a header tbh, it felt natural for it to be. but I guess if it doesnt get used externally then its debatable